### ISSUE TITLE
Allow users to remove original checksum files 

### ISSUE THREAD

**Issue by nkrabben** [amiaopensource/checksumthing#9]
Particularly if users are pulling checksums together into a single file, it would be useful to have the option to remove the old sidecars at the same time.

**Comment by ablwr** [comment]
:crab: Nick "Danger" Krabbenhoeft :crab:

**Comment by nkrabben** [comment]
True, since this is a trojan horse issue for creating bags out of not bags. Maybe the easier solution might be aggregating, running a bag validation, and then

`find path/to/parent -iname "*.md5" -exec rm {} \;`

**Comment by ablwr** [comment]
Where have I seen that script recently? 😆
